src/components/tabs/__internal__/tab-title/tab-title.style.ts

```diff
@@ -31,7 +32,8 @@ const oldFocusStyling = `
 
 const StyledTitleContent = styled.span<StyledTitleContentProps>`
   outline: none;
-  display: inline-block;
+  display: flex;
```

Comment:
**suggestion(non-blocking)**: great idea on using flexbox to automate some of the positioning! I noticed there's still a bit of vertical misalignment, especially with sibling/custom layout content and the validation icon:

![tab-title-vertical-misalignment](https://github.com/user-attachments/assets/0d55018d-22f8-4bf2-ad27-d31871c2c67e)

It looks like this might be caused by some existing code that manually positions the content, including these [overrides for the icons](https://github.com/Sage/carbon/pull/7059/files#diff-9ee2f6b82d25bfaf829417e3554c7b13c06cc70d42993a6bca8f67419c9510afR529-R557). What do you think about removing some of the hardcoded positioning and letting flexbox handle it? It's not absolutely essential, but it could make the tab title more adaptable to custom content.

Reply:
Great suggestion; I think the only issue is the inconsistency between the potential sibling components. In the Storybok example, it uses a small `Pill` and default `Icon` but in reality it could be any other component. I'll have a play with it and see what I can get it to do 👍

Reply:
As discussed, postponing addressing this as if things are going to be re-written in the near future it's probably something we can live with